classes/01class/exercises/c01-git01/ramiray/Readme.md

```diff
@@ -0,0 +1,30 @@
+# c01-git01 - Below commands are used for this exercise . 
+
+1. Merging Exercise:
+```
+git checkout master
```

Comment:
Hi Rami, not sure if you've done a different exercise, the first one is ``level intro3`` which is the third exercise of introduction sequence. Below is the instructions from their website

```
To complete this level, do the following steps:
- Make a new branch called bugFix
- Checkout the bugFix branch with git checkout bugFix
- Commit once
- Go back to master with git checkout
- Commit another time
- Merge the branch bugFix into master with git merge
```

Reply:
nice, marking as resolved